composite/market-closure/package.json

```diff
@@ -31,8 +31,7 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
-    "@chainlink/fcsapi-adapter": "^0.0.3",
-    "@chainlink/finnhub-adapter": "^0.0.3",
+    "axios": "^0.21.1",
```

Comment:
What if we used `@external-adapter/Requester` for HTTP calls? Keeps us in the pattern of having a HTTP "module".

Only drawback I could imagine is if we didn't want retries, but since everything _should_ be instant with cache warming it seems like a good thing to have.

Reply:
So the only reason for pulling in `axios` here was to get the `AxiosRequestConfig` type in dataProvider.ts, as opposed to using the `any` type. We're actually using `Requester` for the HTTP calls.

Do you think we should go for `any` instead of pulling it in as a dependency here?

Reply:
Ohh yeah, that's what I get for commenting before doing a full review :sweat_smile: 

No, I think having the type is always preferred. Maybe instead we change line 23 of @chainlink types to something like
```
  export { AxiosRequestConfig } from 'axios'
```
because every composite adapter will likely need to see the type
or eventually create some composite adapter patterns and types.

Relying on @chainlink/external-adapter's axios dependency is probably better.

Reply:
Yeah I agree, that would be better